great_expectations/validator/validator.py

```diff
@@ -689,12 +693,20 @@ def resolve_validation_graph(
                 validation_graph=graph, metrics=metrics
             )
 
+            # Check to see if the user has disabled progress bars
+            if self._data_context:
+                disable = not self._data_context.progress_bars.is_enabled(
+                    "metric_calculations"
+                )
+            else:
+                disable = len(graph.edges) < 3
+
```

Comment:
This is slightly different because we're calculating a boolean and immediately passing it into the `tdqm` funciton. `disable=disable` is readable to me but let me know if this should be `enable` and `disable=not enable`